corrects DB params in celery and alembic config

Gzing opened the pull request:
### Checklist:

- [x] Code contains relevant tests for the problem you are solving
- [x] Ensure all new and existing tests pass
- [x] Submit to the `develop` branch instead of `master`

### Description:

- Corrected the sql database params in celery app settings and in the alembic migration engine.
- Cleaned up Celery configurations

tyleryasaka commented:
@joshfraser `SQLALCHEMY_DATABASE_URI` is not an environment variable here. It's being set on the app config, and it is actually being assigned the value of `DATABASE_URL `. So we just have the one environment variable to worry about in heroku.

tyleryasaka commented on config/settings.py:
```diff
@@ -39,3 +39,6 @@ def abspath(relative_file_path):
 IPFS_PORT = os.environ.get('IPFS_PORT')
+
+REDIS_URL = os.environ.get('REDIS_URL')
+CELERY_DEBUG = os.environ.get('CELERY_DEBUG', False)
```
1. I don't think this will work. Environment variables read from .env files will be strings, not booleans. So would need to do `os.environ.get('CELERY_DEBUG') == 'True'` or something like that. Or am I mistaken?

2. Can we add `CELERY_DEBUG` to `dev.env`? Can just have it set to false: `CELERY_DEBUG=False`

Gzing replied:
Gotcha! fixing and updating everything. Although celery type coerces this and it will work, this is bad practise and will fix this.

tyleryasaka replied:
Ah ok, didn't know that. 👍

Gzing commented:
@tyleryasaka I had already rebased and pushed this branch onto develop and verified the tests passing with the latest pytest command checking the codestyle as well.
The issue was with web3 library and it's unpinned dependencies, which was caused and fixed a day ago. pushed another commit on this branch upgrading web3 now.
